.github/workflows/ion-test-driver.yml

```
@@ -23,9 +23,14 @@ jobs:
         uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # master
         with:
           repository: amazon-ion/ion-test-driver
-          ref: master
+          ref: fff3a34b70d42c389ebcfd1f5168e550801f8e35
```

```suggestion
          ref: master
```
This should work once we merge https://github.com/amazon-ion/ion-test-driver/pull/30